docs/finalproject/projects.md

```diff
@@ -4,6 +4,8 @@ title: Project Briefings
 sidebar_label: Project Briefings
 ---
 
+You can find here some examples of projects that you could chose to do or that have been done previously, to give you and your team inspiration.
```

Comment:
Who is the "you" addressed here?

Trainees often ask if they can choose their own final project and team, which they cannot. It would be useful to make the intended audience clear here.

Reply:
(Also psst it's choose not chose)

Reply:
In Glasgow, the trainees were allowed to choose/define their final project. I know it's no longer the case so we can change it.

Reply:
To complicate this a little, sorry: I'm not against trainees doing this in Glasgow if they have the support and resources to do it effectively. It's not about them being allowed or not; it's about trainees having the ability to organise, plan, and execute a final project. We have had situations in other regions with no final projects being organised.